config/local.dev.php

```diff
@@ -2,6 +2,11 @@
 
 // Dev environment
 
+// Error reporting
+error_reporting(1);
```

Comment:
This block must be moved into the callback function.

Reply:
about wich callback function do you talk? Do you mean? 
![image](https://user-images.githubusercontent.com/39025147/190844960-550fb897-e5d5-4906-82b7-0967ab8fb407.png)

Reply:
Yes, this callback function as shown in your screenshot.